README.md

```diff
@@ -1 +1,158 @@
-# Portfolio-part-2-Microverse
\ No newline at end of file
+# Portfolio
+Portfolio: setup and mobile first
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [Tech Stack](#tech-stack)
+  - [Key Features](#key-features)
+  - [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 [Portfolio] <a name="about-project"></a>
+
+
+**[Portfolio]** It is a showcase to show my current and future projects.
+
+## 🛠 Built With <a name="built-with-">'Jonathan-Arias'</a>
+
+### Tech Stack <a name="tech-stack">HTML, CSS</a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+  </ul>
+<ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+
+- **[[Descriptive_README_File]**
+- **[Mobile_first_project]**
+- **[Linterns_Best_Practices]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+1. Download the repo or clone it
+
+2. Use Live server extention 
+ID: ritwickdey.LiveServer
+Or run it via GitHub Pages
+### Prerequisites
+
+In order to run this project you need:
+
+Visual Studio or any other editor such as Sublime Text
+A localhost or a github pages set up
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+
+
+### Install
+
+Install this project with:
+
+No installation needed
```

Comment:
- [ ] [Optional] 👍 Great job on adding the "Install" subsection to your README file! However, since there is nothing to install for your project, it is unnecessary to include this section. It can be a bit confusing for visitors to see an section description is "No installation needed". Therefore, I recommend removing the "Install" section from your README file.

By removing the "Install" section, you can enhance the clarity and readability of your README. Additionally, to ensure consistency, remember to remove the "Install" heading from the "Table of Contents" as well.

Keep up the good work! 👍